pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -389,10 +389,13 @@ func convertToEnvoyJwtConfig(jwtRules []*v1beta1.JWTRule, push *model.PushContex
 }
 
 // buildLocalJwks builds local Jwks by fetching the Jwt Public Key from the URL passed if it is empty.
-func buildLocalJwks(jwksURI, jwtPubKey string) *envoy_jwt.JwtProvider_LocalJwks {
+func buildLocalJwks(jwksURI, jwtPubKey string, jwtKeyResolver *model.JwksResolver) *envoy_jwt.JwtProvider_LocalJwks {
```

Comment:
you could just implement buildLocalJwks as a method of JwksResolver so that you don't need to pass the extra argument.

Reply:
Reimplemented as a method, thanks!